thrift-docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
this shouldn't be necessary if they're running in the docker image described in the readme, should it?

Reply:
no, but go development can totally be done locally.

Reply:
so can python, but i put it in the docker env so people don't need to have the right env already installed.